src/main/java/Naruto.java

```diff
@@ -0,0 +1,79 @@
+import java.util.ArrayList;
+
+public class Naruto {
+    private final ArrayList<Item> storedItems;
+
+    public Naruto() {
+        this.storedItems = new ArrayList<>(100);
+    }
+
+    private String addItem(Item item) {
```

Comment:
I like the way you named your methods, by using verbs for actions I can clearly understand what the methods do without having to read the implementation.